samples/apps/bookinfo/bookinfo.yaml

```diff
@@ -82,6 +82,34 @@ spec:
         ports:
         - containerPort: 9080
 ---
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
Should we move this deployment to a separate yaml file (e.g., bookinfo-ratings-v2.yaml)? Starting all versions of all services up front with one yaml file is not such a good idea, especially as we continue to add more versions with additional prereqs.